src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -74,7 +74,7 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
 
             NuGet.Common.Migrations.MigrationRunner.Run();
 
-            if (args.Count() >= 2 && args[0] == "package" && args[1] == "search")
+            if ((args.Count() >= 2 && args[0] == "package" && args[1] == "search") || (args.Any() && args[0] == "config"))
```

Comment:
We should be using length > 0 in all of these. 

Perf isn't critical here, but it's good to be in the habit of using the performant option.